services/web/client/source/class/osparc/desktop/NavigationBar.js

```diff
@@ -113,6 +113,7 @@ qx.Class.define("osparc.desktop.NavigationBar", {
     },
 
     _createChildControlImpl: function(id) {
+      const storeStatics = osparc.store.Store.getInstance().getStatics();
```

Comment:
It is really not a big problem, but I wouldn't put this here as it will get executed a lot, even when it won't be used. Maybe storing it in a class variable in the constructor of the class like this (it would only execute once and be always available within the class)
```javascript
// In the constructor
this.__serverStatics = osparc.store.Store.getInstance().getStatics();
// and then in the childControlImpl
if (this.__serverStatics && this.__serverStatics.manualGuidonManolete) {
   ...
```